app/javascript/components/ResumeGame.js

```diff
@@ -1,70 +1,73 @@
+      .then((res) => {
+        this.props.resumeGame(res);
+      })
+      .catch(() => {});
+  };
 
-    onInputChange = ({ target: { name, value } }) => {
-        this.setState(prevState => ({
-          ...prevState,
-            [name]: value
-        }));
-      };
+  resumeLastGameHandler = () => {
+    const id = localStorage.getItem("gameId");
+    AxiosInstance({ path: `games/${id}`, method: "get" })
+      .then((res) => {
+        this.props.resumeGame(res);
+      })
+      .catch(() => {});
+  };
 
-      resumeGameHandler = (e) => {   
-          e.preventDefault()          
-          AxiosInstance({path:`games/${this.state.id}`, method:'get'} )
-          .then(res => {
-            this.props.resumeGame(res);
-          }).catch(() => {})
-      };
+  render() {
+    return (
+      <>
+        <form className="form-inline ml-4" onSubmit={this.resumeGameHandler}>
+          <div className="input-group mb-2 mr-sm-2">
+            <input
+              type="number"
+              required
+              onChange={this.onInputChange}
+              name="id"
+              className="form-control mb-2 mr-sm-2"
+              placeholder="Enter Game Id"
+            />
+          </div>
 
-      resumeLastGameHandler = () => {  
-            const id = localStorage.getItem('gameId') 
-                AxiosInstance({path:`games/${id}`, method:'get'} )
-                .then(res => {
-                  this.props.resumeGame(res);
-                }).catch(() => {})   
-      };
-
-    render(){
-        return(
-            <>
-                <form className="form-inline ml-4" onSubmit={this.resumeGameHandler}>
-                    
-                    <div className="input-group mb-2 mr-sm-2">
-                    <input type="number" required 
-                        onChange={this.onInputChange} 
-                        name="id" 
-                        className="form-control mb-2 mr-sm-2" 
-                        placeholder="Enter Game Id" />    
-                    </div>
-                
-                    <button href="#" type="submit" id="reset-game">Resume Game</button>
-                </form>
-                <br></br>
-                <br></br>
-                <br></br>
-                <br></br>
-                <br></br>
-                <br></br>
-                <button 
-                    className="mr-4 resumeGame"
-                    id="reset-game"
-                    onClick={() => this.resumeLastGameHandler()} 
-                >Resume Last Game</button>
-
-            </>
-
-        )
-    }
+          <button href="#" type="submit" id="reset-game">
+            Resume Game
+          </button>
+        </form>
+        <br></br>
+        <br></br>
+        <br></br>
+        <br></br>
+        <br></br>
+        <br></br>
+        <button
+          className="mr-4 resumeGame"
+          id="reset-game"
+          onClick={() => this.resumeLastGameHandler()}
```

Comment:
Can we pass in `resumeLastGameHandler` to `onClick` directly? Do we need the additional function around `resumeLastGameHandler`?